docs/user-docs/QUICKSTART.md

```diff
@@ -0,0 +1,174 @@
+# Quick Start Guide
```

Comment:
This file has changed [on documentation repo](https://github.com/TrenchBoot/documentation/blob/master/QUICKSTART.md), it now uses v11, please update.

In addition, please change the title to `Linux Quick Start Guide`.

Reply:
What to do with the Troubleshooting section/document? The section is now different
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/33/commits/2911dcdafb45def359ee307fc44443e9444260fe

Reply:
I added it and rebased to drop empty commit that somehow made it into PR.